tests/php/TestRegistration.php

```
@@ -0,0 +1,189 @@
+		);
+
+		$this->assertTrue(
+			$reflection->hasProperty('unit_fields'),
+			'Bindings class should have unit_fields property'
+		);
+
+		$this->assertTrue(
+			$reflection->hasProperty('unit_types'),
+			'Bindings class should have unit_types property'
+		);
+	}
```

PHPMD (2.15.0)

The method test_custom_fields_logic is not named in camelCase. (undefined)

(CamelCaseMethodName)